Solutions/Cyware/Hunting queries/UnusualNetworkConnectionsToRareExternalDomains.yaml

```diff
@@ -0,0 +1,22 @@
+id: 61c99147-b749-4164-80b1-c4bfa4efa704
+name: Detecting Suspicious PowerShell Command Executions
+description: |
+  'Spot connections to rarely accessed external domains that are present in your watchlist, which could signify data exfiltration attempts or C2 communication.'
+requiredDataConnectors: [
+'DeviceProcessEvents'
+]
+tactics:
+  - Command And Control
+query: |
+DeviceNetworkEvents
+| join kind=inner (
+    _GetWatchlist('CTIX')
+    | where isnotempty(SearchKey)
+) on $left.RemoteUrl == $right.SearchKey
+| project TimeGenerated, RemoteUrl, DeviceName, SearchKey=SearchKey, LastUpdatedTimeUTC
+version: 1.0.0
+metadata:
```

Comment:
Hi @AashiqRamachandran, Metadata is not required for hunting queries within solution